common/src/main/cpp/classes/openxr_fb_passthrough_geometry.cpp

```diff
@@ -49,27 +59,20 @@ void OpenXRFbPassthroughGeometry::set_mesh(const Ref<Mesh> &p_mesh) {
 	mesh = p_mesh;
 
 	if (mesh.is_null()) {
```

Comment:
There was a comment made in https://github.com/godotengine/godot/pull/91247 about how `Ref::is_null()` is different from `!Ref::is_valid()`, example being that if the reference is freed via `queue_free()`, it's no longer valid but it's not `null`.

In the above assignment, we should check if `p_mesh.is_valid()` before assigning it to `mesh`, and set `mesh` to `null` if it's not.

Reply:
The linked issue was specifically about `Callable`. In the case of `Ref`, there is no difference between `is_null()` and `is_valid()` other than them being boolean opposites.

See: https://github.com/godotengine/godot/blob/master/core/object/ref_counted.h#L201

Reply:
I'd assume that `Callable` was also a `Ref` but that's not the case.. thanks for the clarification!